deps/wazuh_testing/wazuh_testing/tools/performance/visualization.py

```diff
@@ -32,7 +32,7 @@ class DataVisualizer:
         base_name (str, optional): base name used to store the images.
     """
     def __init__(self, dataframes, target, compare=False, store_path=gettempdir(), x_ticks_granularity='minutes',
-                 x_ticks_interval=1, base_name=None, columns_path=None):
+                 x_ticks_interval=1, base_name=None, columns_path=None, unify=False):
```

Comment:
```suggestion
                 x_ticks_interval=1, base_name=None, columns_path=None, unify_child_daemon_metrics=False):
```

Reply:
Solved in https://github.com/wazuh/wazuh-qa/pull/5500/commits/4f128638fdbf2976fae270c15f42731b033bddf2